src/http.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable new-cap */
+'use strict';
+
+const _ = require('lodash');
+const RippleAPI = require('./index').RippleAPI;
+const jayson = require('jayson');
+const Promise = require('bluebird');
+
+function createServer(options, httpPort) {
+  const rippleAPI = new RippleAPI(options);
+
+  const functions = _.filter(_.keys(RippleAPI.prototype), k => {

```

Comment:
This should probably be called "methodNames" so that it doesn't sound like an array of references to functions.